lib/generators/templates/spec.erb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+RSpec.describe <%= class_name %> do
+  describe 'call' do
```

Comment:
I would vote for removing the contents of the describe block and replacing them with a simple `pending` message (like in the RSpec model template).